android/app/src/main/java/app/organicmaps/bookmarks/data/Icon.java

```diff
@@ -219,6 +217,16 @@ public int argb()
     return ARGB_COLORS[mColor];
   }
 
+  public static int getColorPosition(int color)
```

Comment:
Track colors are sent and recieved in rgb format not like bookmarks which send index of predefined colors, in order to get the index of predefined colors from rgb i have added a static method here. Since size of color array is not very big i performed linear search. Is is fine or do you have any better idea to get the index from rgb color @rtsisyk

Reply:
@kirylkaveryn how iOS implementation with colors works? Can you please recommend the cleanest solution here that avoids copy-paste and reuses as much code in the core as possible to get/set tracks colors?